pkg/storage/local.go

```diff
@@ -1905,3 +1903,228 @@ func (is *ImageStoreLocal) RunGCRepo(repo string) {
 
 	is.log.Info().Msg(fmt.Sprintf("GC completed for %s", path.Join(is.RootDir(), repo)))
 }
+
+// func (d *MetadataLocalStoreDB) CreateNewUserEntry(userid string) (UserMetadata, error) {
+
+// 	return
+// }
+
+// stars
+const (
+	starredReposKey    = "starredReposKey"
+	bookmarkedReposKey = "bookmarkedReposKey"
+)
+
+func (d *MetadataLocalStoreDB) ToggleStarRepo(userid, reponame string) error {
```

Comment:
MetadataLocalStoreDB implementation should be in a new package, `storage/userdb`, for example.